sdcm/spark_migrator.py

```
@@ -346,14 +348,74 @@ def _upload_runner_script(
+            args = [runner_script_uri]
+            return self.emr_provisioner.add_step(cluster_id=cluster_id, step_name=step_name, jar=jar, args=args)
+
+        # native EMR Spark 4
+        jar = _COMMAND_RUNNER_JAR
+        config_basename = config_s3_path.rsplit("/", 1)[-1]
+        config_ref = config_basename
+        files_arg = config_s3_path
+
+        pre_cmd = ""
+        if deploy_mode == "client":
+            config_ref = f"/tmp/{config_basename}"
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)